deploy/sre-prometheus/100-managed-upgrade-operator.PrometheusRule.yaml

```diff
@@ -70,12 +70,12 @@ spec:
       annotations:
         summary: "Node drain failed in the given time period which is not caused by the PDB"
         description: "node drain takes too long and cannot be finished in the given time period during cluster upgrade"
-    - alert: UpgradeConfigSyncFailureOver1HrSRE
-      expr: upgradeoperator_upgradeconfig_synced > 0
+    - alert: UpgradeConfigSyncFailureOver2HrSRE
```

Comment:
Just a note to please have an ops-sop commit prepared that renames our alert SOP when this goes live.

Reply:
sure, I will make a PR for that